concepts/maps/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Maps are a bultin type in Go that allow to associate a piece of data with a key.",
```

Comment:
Grammar improvement:

Maps are a bultin type in Go that associates values with keys.